megamek/src/megamek/server/Server.java

```diff
@@ -33107,16 +31912,15 @@ public void run() {
         final String METHOD_NAME = "run()";
         Thread currentThread = Thread.currentThread();
         getLogger().info(getClass(), METHOD_NAME, "s: listening for clients...");
-        // HashSet<IConnection> toUpdate = new HashSet<IConnection>();
+        // HashSet<IConnection> toUpdate = new HashSet<IConnection>(); // TODO : Windchild Can this be removed?
```

Comment:
Can this be removed?

Reply:
That looks like it goes with the block of commented out code at the end of the method, so it can go with it.